pages/thc/[processRef]/review.tsx

```diff
@@ -135,6 +173,14 @@ const ReviewPage: NextPage = () => {
             selectedTenantId,
             signatures[selectedTenantId] || ""
           );
+
+          if (!processDatabase.result) {
+            return;
+          }
```

Comment:
We should disable the button if `!processDatabase.result` and combine this check at the top to protect against early submission.

Reply:
Move the check in the callback to the top. We don't want to make any changes if the button should be disabled.